lib/modules/datasource/nuget/__fixtures__/v3_index_no_packagebaseaddress.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
please inline and remove all unused items

Reply:
Not sure what you mean by "inline"—move the contents into `index.spec.ts`?

Reply:
Yes, have it as a string literal inside the test it's used in - it's faster for Jest